src/loader/endpoints/corrections.py

```diff
@@ -0,0 +1,103 @@
+import pandas as pd
+import Levenshtein as lev
+
+main_corrections = pd.DataFrame(
+    {
+        "city_name": ["Januário Cicco"],
+        "state_num_id": [24],
+        "correction_name": ["Boa Saúde"],
+    }
+)
+in_loco_corrections = pd.DataFrame(
+    {

```

Comment:
Does this need to be a DataFrame?

Why not a `.yaml` in a configuration folder?

Also, a better datastructure would be:

```
- original_name: São Valério da Natividade
   correction_name: São Valério
   state_name: Tocantins
- original_name: Santa Teresinha
   correction_name: Santa Terezinha
   state_name: Bahia
```

Is a bit more verbose, but it easier to understand and to manipulate.